docs/solidity-by-example.rst

```diff
@@ -540,6 +540,9 @@ Safe Remote Purchase
         enum State { Created, Locked, Inactive }
         State public state;
 
+        /// Solidity only allows integer arithmetic with even payments.
+        /// An odd number divided by 2 truncates the decimal such that when 
+        /// subsequent re-multiplication by 2 does not equal the same number
```

Comment:
I think this should move above the `require` statement and would try to improve the text. It is not about Solidity allowing even payments, but rather division rounds down and truncates.

```
Division here will truncate if `msg.value` is an odd number. Check via multiplication it wasn't an odd number.
```

Actually it could be replaced with `require((value % 2) == 0)` which is self explanatory.